support-data-rest/src/main/java/org/uniprot/api/support_data/controller/UniProtConfigureController.java

```diff
@@ -23,15 +22,10 @@ public UniProtConfigureController(UniProtConfigureService service) {
     }
 
     @GetMapping("/search_terms")
-    public List<SearchItem> getUniProtSearchTerms() {
+    public List<AdvanceUniProtKBSearchTerm> getUniProtSearchTerms() {
```

Comment:
Does this return the new model to front-end? If so, are they okay with this or could it break things for them (if there are minor changes)?

Reply:
I have added old API with static data